src/applications/gi/containers/LandingPage.jsx

```diff
@@ -54,11 +63,27 @@ export class LandingPage extends React.Component {
   }
 
   render() {
+    const {
+      appGibctLandingPageShowPercent,
```

Comment:
doesn't seem to be a way to control how this gets camel cased by the framework

Reply:
Maybe we can create a map of constants to the flag names and format the names however we like. So we could refer to flags like: `this.props.flags[GIBCT_LANDING_PAGE_SHOW_PERCENT]`.

Reply:
yeah I think I would like to wrap this call so have the option to swap it out for another framework. I like the idea of using constants for the flags. Launch Darkly lets you designate temporary vs permanent flags- I'm not sure how this changes the experience. I'm assuming it will regularly notify you when you have old temporary flags that haven't been evaluated in a while. 

In this [epic](https://github.com/department-of-veterans-affairs/va.gov-team/issues/322#issue-458110119), I laid out some options for how we could organize the flags by platform (required flags available on every app that anyone should be able to use e.g. downtime banners, turning the app off) vs app flags (optional flags managed by the team that require context to understand e.g. a/b tests, new feature rollout). 

Curious what other people think of the naming conventions.